collect_app/src/main/java/org/odk/collect/android/utilities/ImageConverter.java

```diff
@@ -39,9 +39,28 @@
     private ImageConverter() {
     }
 
+    /**
+     * Before proceed with scaling or rotating, make sure existing exif information is stored/restored.
+     * @author Khuong Ninh (khuong.ninh@it-development.com)
+     */
     public static void execute(String imagePath, QuestionWidget questionWidget, Context context) {
```

Comment:
Would you like to add a test? It should be straightforward because we have this function in `ImageConverterTest`:

```
    private void saveTestBitmap(int width, int height, Integer orientation) {
        Bitmap bitmap = Bitmap.createBitmap(width, height, Bitmap.Config.RGB_565);
        FileUtils.saveBitmapToFile(bitmap, TEST_IMAGE_PATH);

        if (orientation != null) {
            try {
                ExifInterface exifInterface = new ExifInterface(TEST_IMAGE_PATH);
                exifInterface.setAttribute(ExifInterface.TAG_ORIENTATION, String.valueOf(orientation));
                exifInterface.saveAttributes();
            } catch (IOException e) {
                Timber.w(e);
            }
        }
    }
```

which we use for testing `ImageConverter`.

Reply:
I have added 1 test to check if exif data is removed after ImageConvert.execute() is invoked.
Please kindly review.